frontend/src/pages/AdminDashboard.tsx

```diff
@@ -19,10 +19,10 @@ export default function AdminDashboard() {
       type: "warning",
       actions: [
         {
-          text: t("continue"),
+          text: t("delete"),
           type: "ghost",
           fn: () => {
-            userPurge(authUser?.uid);
+            userPurge(authUser!.uid);
```

Comment:
@jnorlando `!` Fixes the typescript error, this is generally a dangerous move as it ignores the fact that `authUser` could be null.

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#non-null-assertion-operator